src/content/whats-new/2022/03/whats-new-03-28-futurestack-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'FutureStack is back—in-person at the Cosmopolitan of Las Vegas' 
```

Comment:
@Alanjbaptista this 'back-in-person' is a little awkward to me. Maybe something like: "FutureStack is back and in-person at the...

Reply:
Updating the source file.